randovania.spec

```diff
@@ -57,6 +61,25 @@ a = Analysis(['randovania/__main__.py', 'randovania/cli/__init__.py'],
              win_private_assemblies=False,
              cipher=block_cipher,
              noarchive=False)
+
+
+def calculate_hash(file_name: str):
+    print(f"Hashing {file_name}")
+    with open(file_name, "rb") as f:
+        return hashlib.sha256(f.read()).hexdigest()
+
+
+frozen_file_list = pathlib.Path("randovania").joinpath("data", "frozen_file_list.json")
+frozen_file_list.write_text(json.dumps({
+    entry[0]: calculate_hash(entry[1])
+    for entry in a.datas + a.binaries
```

Comment:
this will raise failures for certain files for which we support user modification. most notably configuration.json, but also autotracker layouts

Reply:
I would say that it changing these files has it's use cases and work, but I wouldn't say it's a recommended workflow.

And if you changed these and then clicked to see if there's anything wrong in your installation, it certainly should catch these.